### ISSUE TITLE
elasticsearch not starting with plugin configuration

### ISSUE THREAD

**Issue by nreese** [sscarduzio/elasticsearch-readonlyrest-plugin#28]
I have successfully installed readonlyrest plugin version 1.7 into an elasticsearch 2.2 instance. I can start elasticsearch and verify that the line 'plugins[readonlyrest]' is contained in the elasticsearch log file.

However, if I start to configure readonlyrest plugin in /etc/elasticsearch/elasticsearch.yml file, elasticsearch does not start and elasticsearch.log file never gets created. Below is my configuration. 

```
readonlyrest:
    enable: true
    access_control_rules:

    - name: Accept only GETs with empty body from other host
      type: allow
      methods: [GET]
      maxBodyLength: 0
```
What am I doing wrong? Is there a way to get more verbose logging?

Thanks,

Nathan

**Comment by sscarduzio** [comment]
Wow that's pretty extreme! I'll try to reproduce it tomorrow.

**Comment by nreese** [comment]
I saw this [stackoverflow question] (http://stackoverflow.com/questions/27273998/why-elasticsearch-wont-run-on-ubuntu-14-04) where the first answer states 
```
Elasticsearch service init script doesn't print any error information on console or log file when it fails to startup, 
instead it ridiculously shows [OK].
``` 

I followed the solution to manually start elasticsearch in the hopes of getting an error message. To my amazement, elasticsearch started with the readonlyrest configurations. I was able to verify that the plugin was properly initialized by hitting http://ip:9200/_nodes?plugins=true and seeing readonlyrest plugin with the access control rule.

The confusing part is that I still can not start elasticsearch as a service using /etc/init.d/elasticsearch with readonlyrest configurations.

So maybe the issue title should be  changed to 'When starting elasticsearch as a service, elasticsearch not starting with plugin configuration'
